cmd/inspect/malware.go

```diff
@@ -80,6 +81,19 @@ func executeMalwareAnalysis() error {
 
 	packageVersion := purl.PackageVersion()
 
+	// If package version is empty or latest replace it with actual literal latest version
+	// Reference: https://github.com/safedep/vet/issues/446
+	if packageVersion.GetVersion() == "" || packageVersion.GetVersion() == "latest" {
+		ui.PrintMsg("Resolving package version")
+		version, err := common.ResolvePackageLatestVersion(purl.Name(), purl.Ecosystem())
+		if err != nil {
+			return fmt.Errorf("failed to resolve package version: %v\n\nTry with specified explicit version", err)
```

Comment:
```suggestion
			return fmt.Errorf("failed to resolve package latest version: %v", err)
```

Reply:
Can't use `\n\n` in error message. Don't assume they will be printed to console only. They may be Iog files or streamed to some log server. The `\n\n` breaks automated log analysis.